tests/admin.test.ts

```diff
@@ -182,3 +184,105 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
```

Comment:
Thanks for writing these tests! While these are technically admin activities, I'd say we should create a new test file called `event.test.ts` and have these tests exist there. We tend to correlate test files with the exact controller they reference, so `EventController` tests should be in `event.test.ts` instead of `admin.test.ts`.